docs/config/index.md

```diff
@@ -318,6 +318,27 @@ Isolate environment for each test file
 
 Coverage options
 
+### testNamePattern
+
+- **Type** `string | RegExp`
+
+Run tests with full names matching the pattern.
+If you add `do not run` to this property, tests containing the word `do not run` in the test name will be skipped.
```

Comment:
I feel like the opposite will happen? Only tests containing "do not run" will run, no?